templates/web/fixmystreet.com/around/postcode_form.html

```diff
@@ -0,0 +1,34 @@
+<div id="front-main">
+    <div id="front-main-container">
+        [% UNLESS possible_location_matches %]
+            [% INCLUDE 'around/intro.html' %]
+        [% END %]
+
+        [%
+            question = c.cobrand.enter_postcode_text || loc('Enter a nearby street name and area');
+        %]
+
+        <form action="[% c.uri_for('/around') %]" method="get" name="postcodeForm" class="js-geolocate postcode-form-test">
+            <label for="pc">[% question %]:</label>
+            <input type="text" name="pc" value="[% pc | html %]" id="pc" size="10" maxlength="200" placeholder="[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]" class="postcode-form-test__postcode">
+            <a href="[% c.uri_for('/around') %]" id="geolocate_link" class="btn btn--geolocate">[% loc('or locate me automatically') %]</a>
+
+            [%# Form will be submitted via this button when user presses Enter key %]
+            <button type="submit" class="visuallyhidden" aria-hidden="true" tabindex="-1">Go</button>
+
+            <div class="postcode-form-test__buttons">
+                [%# We can now segment users in Google Analytics by looking at "intent" query string param %]
+                <button type="submit" name="intent" value="report" class="btn">[% loc('Report a new problem') %]</button>
+                <button type="submit" name="intent" value="browse" class="btn">[% loc('Show problems') %]</button>
```

Comment:
At narrow width these go onto two lines and mean the instructions are pushed out of sight - can we keep them on one line perhap?

Reply:
Good suggestion :+1: Turns out flexbox makes this one a doozy. Done in `fixmystreet.com/base.scss` -> `.postcode-form-test__buttons`